src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -325,6 +325,11 @@
             nativeEditor.on('change', saveRichText);
             nativeEditor.on('customUpdate', saveRichText);
             nativeEditor.on('editorInteraction', saveRichText);
+            nativeEditor.on('paste', () => {
+                setTimeout(() => {
```

Comment:
Timeout is only for that it should be invoked `after` not `on`?
Maybe you should use the `afterPaste` event ref: https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-afterPaste

Reply:
Thanks, I'll check that and adjust both PR's eventually.

Reply:
Applied in https://github.com/ezsystems/ezplatform-admin-ui/pull/1245/commits/f8b36ca428a8d358d9712dec4abf149a21934613.